metadata.yaml

```diff
@@ -6,6 +6,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
```

Comment:
Nice catch!

@cahillsf Usually (1.5 & 1.6) we did this as part of "Prepare main branch for development of the new release".

Looks like we missed it for v1.7, but it also looks like it is not documented in release-tasks

Reply:
ah, it is in [the tasks](https://docs.google.com/document/d/157pNz7xpORkiMVYFyDIMJIRShWxVBxiThF20ymlWaHc/edit?pli=1) but i missed it this time.  thanks for fixing!

Reply:
Okay I don't know what this google doc is :) (I don't have access)

Shouldn't it be documented in the publicly accessible release-tasks.md? (I didn't find it there, but maybe I missed it)

Reply:
Ah I dropped the wrong link, just fixed
> Add the new release to the root level metadata.yaml

Reply:
Ah :D 

Okay somehow I missed it, but it's there
> Add the new release to the root level metadata.yaml